### PR TITLE
[feat][DTC-5411] updated jobs runner to be able to use temporal

### PR DESCRIPTION
**Pull request by Sthakur27**
Updating env vars for retool onpremise to allow jobs runner to access temporal and turn into a internal onprem temporal worker.

https://linear.app/retool/issue/DTC-5411/update-docker-images-with-new-jobs-runner-env-vars

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ services:
       - DB_CONNECTOR_PORT=3002
       - DB_SSH_CONNECTOR_HOST=http://db-ssh-connector
       - DB_SSH_CONNECTOR_PORT=3002
+      - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_HOST=temporal
+      - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_PORT=7233
+      - WORKFLOW_BACKEND_HOST=http://api:3000
```
Comment (avimoondra) [inline docker-compose.yml:16]: why is this included here?  i think the comments from L17 to L25 should be pasted into the jobs-runner service definition?

**Review by avimoondra — CHANGES_REQUESTED** [review]
LGTM - any docs changes that should be paired with this?

File: docker-compose.yml (hunk 1 above)
Comment (Sthakur27) [reply docker-compose.yml]: good catch, updated

**Review by alex-stabile — CHANGES_REQUESTED** [review]
Why are we making these changes? I think we need to align with the rest of DTC on what the plan is here and why we're making these changes.

I also don't think we should "convert" jobs runner into a temporal worker. I like @roberto-retool 's suggestion [here](https://retooled.slack.com/archives/C05C6BYRR4G/p1697227243777439?thread_ts=1697225379.965519&cid=C05C6BYRR4G) about adding source control to the temporal worker.

I don't see why we would update the public `on-premise` repo until we've validated our new setup with a customer
